docs/manual/job-options.md

```diff
@@ -476,6 +476,7 @@ Option model providers are configured on a per-Option basis (where a Job may hav
    - OR, an array of Maps, each with two entries, `name` and `value`.
 4. By default, the HTTP(S) response must include the `application/json` content type in the header.
    In case this cannot be controlled, the attribute `project.jobs.disableRemoteOptionJsonCheck` can be set to `true` in the project settings.
+5. On HTTP(S) unsafe characters should be avoided within a URL, they may cause errors since characters will be inconsistent after being encoded.   
```

Comment:
Do we have a list of "unsafe" characters or examples?

Reply:
I added a list of unsafe characters.